### PR TITLE
#187 NewAdminBundle: Added warning for non-playable mmobjs.

### PR DESCRIPTION
**Pull request by Alfro**
This warning is meant to give a more intuitive view of which mmobjs will actually be played on the playlist and which ones won't.

Note: Assign issue #187 for QA when this is merged.

### CODE REVIEW

File: src/Pumukit/NewAdminBundle/Twig/PumukitAdminExtension.php [hunk 1]
```diff
@@ -556,4 +557,22 @@ private function countMmobjsWithTag($series, $tagCod)
...
+     * Returns a boolean with whether the mmobj will be played on a playlist.
+     *
+     * @param MultimediaObject $mmobj
+     *
+     * @return boolean
+     */
+    public function isPlaylistPlayable($mmobj)
+    {
+        $broadcast = $mmobj->getEmbeddedBroadcast();
+        if ($broadcast->getType() != EmbeddedBroadcast::TYPE_PUBLIC
```
Comment (rubenrua) [inline src/Pumukit/NewAdminBundle/Twig/PumukitAdminExtension.php]: `$broadcast` can be null (if null is public).

File: src/Pumukit/NewAdminBundle/Twig/PumukitAdminExtension.php [hunk 2]
```diff
@@ -556,4 +557,22 @@ private function countMmobjsWithTag($series, $tagCod)
...
+
+
+    /**
+     * Returns a boolean with whether the mmobj will be played on a playlist.
+     *
+     * @param MultimediaObject $mmobj
+     *
+     * @return boolean
+     */
+    public function isPlaylistPlayable($mmobj)
```
Comment (rubenrua) [inline src/Pumukit/NewAdminBundle/Twig/PumukitAdminExtension.php]: This logic must be in a service.

**Comment by Alfro** [comment]
@rubenrua Take another look.

**Comment by rubenrua** [comment]
Thank you.

**Pull request merged**